rust/datafusion/tests/sql.rs

```diff
@@ -517,9 +518,17 @@ fn register_aggregate_csv_by_sql(ctx: &mut ExecutionContext) {
     WITH HEADER ROW
     LOCATION '{}/csv/aggregate_test_100.csv'
     ",
-        testdata
-    ))
-    .unwrap();
+            testdata
+        ))
+        .expect("Creating dataframe for CREATE EXTERNAL TABLE");
+
+    // Mimic the CLI and execute the resulting plan -- even though it
+    // is effectively a no-op (returns zero rows)
+    let results = df.collect().expect("Executing CREATE EXTERNAL TABLE");
```

Comment:
Prior to the code change, the tests now fail in the same way as reported in ARROW-9889:

```
---- csv_query_create_external_table stdout ----
thread 'csv_query_create_external_table' panicked at 'Executing CREATE EXTERNAL TABLE: General("Unsupported logical plan variant")', datafusion/tests/sql.rs:526:19
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
```